operator/v1/zz_generated.crd-manifests/0000_12_etcd_01_etcds-CustomNoUpgrade.crd.yaml

```diff
@@ -40,6 +40,18 @@ spec:
             type: object
           spec:
             properties:
+              backendQuotaGiB:
+                default: 8
+                description: backendQuotaGiB sets the etcd backend storage size limit
+                  in gibibytes. The value should be an integer not less than 8 and
+                  not more than 32. When not specified, the default value is 8.
+                format: int64
+                maximum: 32
+                minimum: 8
+                type: integer
+                x-kubernetes-validations:
+                - message: can't decrease database size
```

Comment:
Might suggest we reword this, maybe `etcd backendQuotaGiB may not be decreased`, prefer to use full sentences and no abbreviations in text as this is user facing

Reply:
done 👍🏽